Automattic/course
# Add hero homepage pattern

Implements https://github.com/Automattic/sensei/issues/6072

## Changes proposed

Added a new pattern name Hero for the homepage.

## Testing instructions

- Create a new page
- In the editor, change the template of the page to Homepage
- Now clear if there's any block on the page and click on the inserter
- Click on patterns Tab
- Click on the dropdown for pattern categories and select "Sensei LMS" from there
- Make sure the new pattern is there
- Click on the pattern and make sure the design matches

## Screenshot

### Desktop
https://user-images.githubusercontent.com/6820724/196927773-17aaf93d-b17b-478a-bf2f-9152d9a37ed7.mov

### Mobile

https://user-images.githubusercontent.com/6820724/196928540-f4848a8f-bbc1-4b76-99f6-15d731ba1f25.mov
